extensions/README.md

```diff
@@ -59,6 +59,13 @@ Generally, all new extensions should start out as a beta.
 - Once you've successfully beta tested your new extension, you can open new PR to make your extension live!
 - Simply move the extension's slug out of the beta array and into the production array in `extensions/index.json`.
 
+### Experimental Extensions
+
+We also offer an "experimental" state for extensions. Those extensions will be made available to anyone:
+- Running the Gutenberg plugin, or;
+- Running the FSE plugin, or;
```

Comment:
I think it would be better to leave for FSE plugin to set this via the filter, and Jetpack would be kind of agnostic about it. What do you think?

Reply:
Yeah, I'd be fine with that. I'm happy to remove Gutenberg as well if needed.

Reply:
I think that would make sense, yeah.

Reply:
Done in ce966f401f798019992cb7e3cdb3e8c35074113d